src/mWater_collate/mWater_pull_collate.Rmd

```diff
@@ -52,7 +52,7 @@ all_notes_cleaned <- all_notes%>%
     #start and end dt comes in as UTC -> to MST
         start_dt = with_tz(ymd_hms(start_dt), tz = "America/Denver"),
         end_dt = with_tz(ymd_hms(end_dt), tz = "America/Denver"),
-         date = as.Date(start_dt),
+         date = as.Date(start_dt, tz = "America/Denver"),
```

Comment:
is this going to always be in MST? Important to put in MST due to how API data comes in....